.markdownlint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "code-block-style": false
```

Comment:
oh wait, can we make this fenced? or why not?

Reply:
Having this be `fenced` messes with places which have to be indented, like inside an admonition

Reply:
also, squashed

Reply:
There's some advice in https://github.com/DavidAnson/markdownlint/issues/207 about how to support admonitions. Any interest in making that work in this PR?